sql/catalyst/src/main/java/org/apache/spark/sql/connector/util/V2ExpressionSQLBuilder.java

```diff
@@ -38,6 +39,11 @@ public String build(Expression expr) {
       GeneralScalarExpression e = (GeneralScalarExpression) expr;
       String name = e.name();
       switch (name) {
+        case "IN": {
```

Comment:
nit: do we really need to wrap the code with `{...}`?

Reply:
The scope of variables span the entire switch.